docs/html/pmc-manual.html

```diff
@@ -201,22 +201,36 @@
 <pre class="programlisting">
 Number of stores not made persistent: 1
 Stores not made persistent properly:
-[0]    at 0x........: main (flush_check.c:27)
-	Address: 0x........	size: 8	state: COMMITTED
+[0]    at 0x1234: main (flush_check.c:27)
```

Comment:
The first address is the instruction address, and the second is data address, right?
So, I would use other values to make it more clear.